### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by amirunxiayang** [aimagelab/show-control-and-tell#35]
why loss = loss_cap + 4*loss_gate,i can not understand the 4
![image](https://user-images.githubusercontent.com/72070010/94533446-b4094b00-0271-11eb-92b9-2fb993a28c51.png)
